woof-code/rootfs-packages/acpid_busybox/etc/acpi/actions/suspend.sh

```diff
@@ -4,6 +4,29 @@
 # 23apr12 fix was not suspend from acpi_poweroff.sh
 #20140526 shinobar: avoid multiple run
 #20140629 shinobar: ACPI_CONFIG
+case "$(awk '{print $2}' /proc/acpi/button/lid/LID0/state)" in
+open)
+  [ "$DISTRO_TARGETARCH" = "x86" ] && exit 0
+  # turn on the internal display on lid open
+  if [ -n "$WAYLAND_DISPLAY" ]; then
+    wlr-randr --output eDP-1 --on
+  elif [ -n "$DISPLAY" -a -z "`pidof -s xlock`" ]; then
+    xrandr --output eDP-1 --on
```

Comment:
- [ ] Test under X.Org
- [ ] Verify that `eDP-1` is a good assumption on various old laptops